app/controllers/coronavirus_local_restrictions_controller.rb

```diff
@@ -14,7 +14,7 @@ def show
 
     if @search.blank_postcode? || @search.invalid_postcode?
       render
-    elsif @search.no_information?
+    elsif @search.no_restriction_information_england_postcode? || @search.no_information?
```

Comment:
Nice one solving the problem here 🎉 

I feel like there's an element of a fight here to fit this in which makes it a little tricky to understand. I don't think that is too big a deal as I've been working on a wider refactor that I think makes it easier:

```
    if @search.blank_postcode? || @search.invalid_postcode?
      render
    elsif @search.devolved_nation_result
      @result = @search.devolved_nation_result

      render :devolved_nation_result
    elsif @search.england_result
      expires_in(restriction_expiry(@search.england_result), public: true)
      @result = @search.england_result

      render :england_result
    else
      render :no_information
    end
```

Reply:
nice!